fixtures/flight-vite/package.json

```diff
@@ -0,0 +1,32 @@
+{
+	"private": true,
+	"type": "module",
+	"scripts": {
+		"build": "vite build --app",
+		"dev": "vite dev",
+		"format": "npx biome format --write",
+		"start": "cross-env NODE_ENV=production node serve.js dist/prerender/entry.prerender.js dist/server/entry.server.js dist/browser",
+		"test": "playwright test"
+	},
+	"dependencies": {
+		"@jacob-ebey/react-server-dom-vite": "19.0.0-experimental.14",
+		"@jacob-ebey/vite-react-server-dom": "0.0.6",
```

Comment:
Ideally it should be possible to install only `react-server-dom-vite` (from React) and the official `vite` stuff. As it is right now you need to install this third party thing.

It also makes it hard for us to make suggestions for how to improve the layering between the packages (and the dependency injection protocol). E.g. we can't change anything in React without also changing this package in lock step.

Can this package just be inlined into the React repo?

Either into the fixture for things related to just wiring up the host, or if they're related to the vite plugin itself then it can move into `react-server-dom-vite`.

For example, all that virtual file and manifest generating stuff seems like it should be living inside `react-server-dom-vite` rather than every user have to reimplement in different ways.

Reply:
The implementation inside `react-server-dom-vite` can be completely different between development and production if needed.